FirebaseABTesting/Sources/FIRExperimentController.m

```diff
@@ -195,6 +195,19 @@ - (void)updateExperimentsWithServiceOrigin:(NSString *)origin
   });
 }
 
+- (void)updateExperimentsWithServiceOrigin:(NSString *)origin
+                                    events:(FIRLifecycleEvents *)events
+                                    policy:(ABTExperimentPayload_ExperimentOverflowPolicy)policy
+                             lastStartTime:(NSTimeInterval)lastStartTime
+                                  payloads:(NSArray<NSData *> *)payloads {
+  [self updateExperimentsWithServiceOrigin:origin
+                                    events:events
+                                    policy:policy
+                             lastStartTime:lastStartTime
+                                  payloads:payloads
+                         completionHandler:nil];
```

Comment:
nit: Consider FIRLogging the error, so users who tab autocomplete into using this method or use other SDKs that call this method don't have the error omitted from logs.

Reply:
Users will get the error logged regardless of whether there's a completion handler (see line 230 of `FIRExperimentController.m` which is prior to the completionHandler nil check).